### PR TITLE
Feat build script

### PR DESCRIPTION
**Pull request by vishnu-ravi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "yelda-webchat",
-  "version": "1.0.47",
+  "version": "1.0.49",
```
Comment (codebynao) [inline package.json:3]: is there a reason to skip 1.0.48? Or was it a typo?
I saw that you already published this version, so it is to late to update anyway

File: uploadBuildToS3.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+  return await executeCommand('git rev-parse --abbrev-ref HEAD')
+}
+
+/**
+ * Updates the NPM version & git commit added with the updated version
+ *  @return {Promise<string>} stdout
+ */
+const updateNPMVersion = async () => {
+  console.info('Updating npm version ...')
+  return await executeCommand('npm -no-git-tag-version version patch')
```
Comment (codebynao) [inline uploadBuildToS3.js:75]: Question here
The current version is 1.0.49
Using this command, the version will be updated to 1.0.50 right ?
But what if we want to update to 1.1.0 or even to 2.0.0 ?

I think we should add an option to the script comment to choose the update type ? This way we could dynamically execute the command:
`npm -no-git-tag-version version patch` OR `npm -no-git-tag-version version minor` OR `npm -no-git-tag-version version major` ?
But we could keep `patch` as default if no update type as been precised.

What do you think ? @meelie

File: package.json (hunk 1 above)
Comment (vishnu-ravi) [reply package.json:3]: I have published this 1.0.48 version to test the script

Comment (codebynao) [reply package.json:3]: Oh okay!

File: uploadBuildToS3.js (hunk 2 above)
Comment (meelie) [reply uploadBuildToS3.js:75]: Good feedback.
Can be done when we will need it ;)

**Pull request merged**